examples/pipeline_tacotron2/loss/__init__.py

```diff
@@ -0,0 +1 @@
+from .loss_function import Tacotron2Loss
```

Comment:
Please add a new line.

Reply:
I've removed `examples/pipeline_tacotron2/loss` directory and moved everything to `examples/pipeline_tacotron2/loss.py` as the tests are moved to `test/torchaudio_unittest/example`.